CHANGELOG.md

```diff
@@ -1,7 +1,34 @@
 # 0.6.0
 
+## Bug fixes
+
 * Fix panic when using incorrect globs f6or file and type selections.
 
+## New feautures
+
+* Also replace `Train-Case` (alo known as `Http-Case`) variants when using `--subvert`
+
+* The core functionnality has been re-written
+
+The output changes lightly:
+
+```
+# Using version < 0.6
+Patching foo/bar.js
+--- old is old
++++ new is new
+```
+
+
+```
+# Using version >= 0.6
+foo/bar.js 3: old is old
```

Comment:
If of any importance: I didn't understand from the single line example that 3: was a line number.

Reply:
I think it's important ;) I've added some clarification